### DISCUSSION TITLE
Support for multiple wsgi apps

### DISCUSSION THREAD

**Discussion by bitjep** [logandk/serverless-wsgi#243] [Q&A]
Hi, I would like to deploy multiple WSGI applications using Serverless Framework. I've seen that the feature was not available on 2019 (#102 ). I wonder if it may be available now.

**Comment by adamoverbeeke** [comment]
Could you just create an environment variable and assign it to your lambda function? You could use the variable in the server.app, then 
 register the api endpoints based on the environment. This assumes that you have split your api calls into separate controllers.
